aptos-move/aptos-global-cache-manager/src/config.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+/// Configuration used for global caches.
+pub struct GlobalCacheConfig {
+    /// If true, when global caches are empty, Aptos framework is prefetched into module cache.
+    pub prefetch_framework_code: bool,
+    /// The maximum size of module cache. If module cache exceeds this capacity, it should be
```

Comment:
is it in bytes? can point out here or in the name.

Reply:
No, in entries. We can use bytes here as well, but module size is bounded so we always now that `cache size < number entries * max size`

Reply:
cool, let's explain that then :)

Reply:
Switched to bytes, and added more comments around that